packages/fetch-core/src/withFetchRedirectionOn401.ts

```diff
@@ -2,9 +2,9 @@
 import { withRouter } from '@axa-fr/react-oidc-core';
 import { wrapAuthenticating } from './withFetchRedirectionOn403';
 
-export const fetchWithRedirectionOn401 = (fetch, history) => async (
-  url,
-  options = { method: 'GET' }
+export const fetchWithRedirectionOn401 = (fetch: typeof window.fetch, history: ReactOidcHistory) => async (
+  url: RequestInfo,
+  options: RequestInit = { method: 'GET' }
 ) => {
   const response = await fetch(url, options);
   if (response.status === 401) {
```

Comment:
Do you think we will be able to inject native dom history object here?

Reply:
I have to check this. The `History` [definition](https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L9240) does not seems to have `push` function